ocaml/idl/datamodel_lifecycle.ml

```diff
@@ -30,7 +30,7 @@ let prototyped_of_field = function
   | "VTPM", "persistence_backend" ->
       Some "22.26.0"
   | "host", "numa_affinity_policy" ->
-      Some "23.32.0-next"
```

Comment:
Please do not commit changes in this file, it's being done in https://github.com/xapi-project/xen-api/pull/5370

Reply:
I have deleted that.

Reply:
I'll check other `List.mem` usages tomorrow.